dev-python/html5lib/html5lib-1.1.ebuild

```diff
@@ -18,6 +18,10 @@ KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~mips ppc ppc64 ~riscv ~s390 sparc x
 IUSE="test"
 RESTRICT+=" !test? ( test )"
 
+PATCHES=(
+	"${FILESDIR}"/${P}-pytest6.patch
+)
+
 RDEPEND=">=dev-python/six-1.9[${PYTHON_USEDEP}]
 	dev-python/webencodings[${PYTHON_USEDEP}]"
 BDEPEND="
```

Comment:
Drop adding RDEPEND in test as det does it for us

Reply:
done